pages/data.py

```diff
@@ -176,6 +188,24 @@ def update_dropdowns_trips(n_clicks, button_label):
     return hidden_col, button_label
 
 
+@callback(
+    Output('uuids-table', 'data'), # Output all data in the uuid-table
+    Output('load-data', 'style'),
+    Input('load-data', 'n_clicks'), #number of clicks on the button
+    Input('store-uuids', 'data'),
+    State('uuids-table', 'data'), #State representing the current data on table
+    prevent_initial_call=True
```

Comment:
Can you please explain what `prevent_initial_call` does?

Reply:
It prevents callback from being triggered when the app is first loaded.